scripts/traas.sh

```diff
@@ -81,21 +81,9 @@ for PROJFULLREF in $CI_REFS ; do
     echo ${change[*]}
     project=${change[0]}
     change=${change[1]}
```

Comment:
LGTM, we do not fail if [[ $project =~ [openstack-infra/tripleo-ci|openstack/tripleo-quickstart|openstack/tripleo-quickstart-extras] ]]; then
    curl https://review.openstack.org/changes/${change}/revisions/current/patch?download | \
    base64 -d | \
    sudo patch -f -d ./${project##*/} -p1
else
  echo "Not proper repo"
  exit 1
fi

but I would like to keep it as simple as possible